app/scenes/right_menu_drawer/right_menu_drawer.js

```diff
@@ -7,7 +7,8 @@ import {
     StyleSheet,
     Text,
     Platform,
-    View
+    View,
+    Linking
```

Comment:
on IOS don't you need to link `RCTLlinking` as described here https://facebook.github.io/react-native/docs/linking.html ?